src/js/fetchHomeTimeline.js

```diff
@@ -1,6 +1,6 @@
-const fetchTimeline = () => {
+const fetchHomeTimeline = () => {
 
-    let emptyPromise = new Promise((resolve) => resolve([]));
+    let emptyPromise = new Promise((resolve) => resolve(""));
```

Comment:
this looks odd.  Why return back an emptyPromise?

Reply:
There are times the API will return an empty array and this isn't always an error. I'm controlling for when an error occurs by sending a response that is not an array